Utils.py

```diff
@@ -84,3 +89,12 @@ def get(self):
         self.check.remove(item)
         return item
 
+# ============== Logging Setup
+
+
+def SetLoggerUname(uname):
+    LoggingConfig.LOGGINGCONFIG['handlers']['handler_file']['filename'] = './logs/{}_{}.log'.format(uname, time.strftime("%y%m%d%H%M%S", time.localtime()))
+    logging.config.dictConfig(LoggingConfig.LOGGINGCONFIG)
+    return logging.getLogger('RunOneSim')
```

Comment:
I did it in a slightly different way, I just wanted to ask if it's valid, otherwise will change to what you asked for. I just thought this way we have less log related lines in RunOneSim and we also don't have an abstraction violation in Utils.